operators/src/source/csv.rs

```diff
@@ -131,13 +133,10 @@ pub struct CsvSourceStream {
     parameters: CsvSourceParameters,
     bbox: BoundingBox2D,
     chunk_size: usize,
-    state: Arc<Mutex<CsvSourceStreamState>>,
-}
-
-pub struct CsvSourceStreamState {
-    csv_reader: ReaderState,
+    reader_state: Arc<Mutex<ReaderState>>,
+    thread_is_computing: Arc<AtomicBool>,
```

Comment:
can't you use AtomicBool directly?

Reply:
No, things that I put into the thread must be `'static`.